pkg/operator/v311_00_assets/bindata.go

```diff
@@ -344,6 +346,66 @@ func v3110KubeSchedulerPodYaml() (*asset, error) {
 	return a, nil
 }
 
+var _v3110KubeSchedulerPolicyconfigmapRoleYaml = []byte(`#As of now, system:kube-scheduler role cannot list configmaps from openshift-kube-scheduler namespace. So, creating a role.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: system:openshift:sa-listing-configmaps
+  namespace: openshift-kube-scheduler
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - get
+`)
+
+func v3110KubeSchedulerPolicyconfigmapRoleYamlBytes() ([]byte, error) {
+	return _v3110KubeSchedulerPolicyconfigmapRoleYaml, nil
+}
+
+func v3110KubeSchedulerPolicyconfigmapRoleYaml() (*asset, error) {
+	bytes, err := v3110KubeSchedulerPolicyconfigmapRoleYamlBytes()
+	if err != nil {
+		return nil, err
+	}
+
+	info := bindataFileInfo{name: "v3.11.0/kube-scheduler/policyconfigmap-role.yaml", size: 0, mode: os.FileMode(0), modTime: time.Unix(0, 0)}
+	a := &asset{bytes: bytes, info: info}
+	return a, nil
+}
+
+var _v3110KubeSchedulerPolicyconfigmapRolebindingYaml = []byte(`# As of now, system:kube-scheduler role cannot list configmaps from openshift-kube-scheduler namespace. So, creating a role.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  namespace: openshift-kube-scheduler
+  name: system:openshift:sa-listing-configmaps
+roleRef:
+  kind: Role
+  name: system:openshift:sa-listing-configmaps
+subjects:
+- kind: ServiceAccount
```

Comment:
The generated data is wrong. I missed two commits which is causing this PR to fail.

Note to myself: Add verify-generated bindata..

Reply:
/cc @sjenning